mongo/assets/configuration/spec.yaml

```
@@ -29,6 +29,10 @@ files:
 
         Do not specify more than one host here unless you are connecting to a mongos instance
         and you want to balance requests performed by the agent to multiple mongos instances.
+        
+        If you need to monitor multiple mongos instances and are using version 7.39 or later of the Agent, configure the integration to disable directConnection, which is enabled by default starting in 4.0.2. 
```

```suggestion
        If you need to monitor multiple mongodb instances and are using version 7.39 or later of the Agent, configure the integration to disable directConnection, which is enabled by default starting in 4.0.2. 
```

BTW @buraizu is this the official way to call mongo in our docs?